### ISSUE TITLE
Accordion: Multiple bad reader items

### ISSUE THREAD

**Issue by EldRoos** [sebgroup/green#943]
### Bug already reported ?

- [X] I confirm that I have checked if the bug already has been reported

### Component name

Accordion

### Description

Missing icon <title> so readers do not "see" the icon. Hide with  <svg aria-hidden="true">.

Header and sub-header looks identical in code. So you can't see where the sub-header is.

Only says expanded when opening it, no way of showing/hearing the inner information.
